envoy/config/filter/http/jwt_authn/v2/README.md

```diff
@@ -0,0 +1,84 @@
+# JWT Authentication envoy HTTP filter config
+
+## Overview
+
+1. The proto file in this folder defines a HTTP filter config for "jwt_authn" filter. This filter will be implemented in Envoy.
```

Comment:
`This filter will be implemented in Envoy.` is not needed.

Reply:
Agree with @lizan, please remove this sentence.